Jenkinsfile

```diff
@@ -84,46 +84,47 @@ timeout(time: 2, unit: 'HOURS') {
             }
             stage('Test Docker app') {
                 checkout scm
-                sh 'git config --global user.email travis@camptocamp.com'
-                sh 'git config --global user.name Travis'
                 sh 'rm -rf ${HOME}/workspace/testgeomapfish'
                 sh 'docker build --tag=camptocamp/testgeomapfish-external-db:latest docker/test-external-db'
                 sh 'travis/create-new-project.sh ${HOME}/workspace'
-                sh 'travis/run-on.sh ${HOME}/workspace/testgeomapfish/ ./docker-run travis/empty-make --makefile=travis.mk help'
-                sh 'travis/run-on.sh ${HOME}/workspace/testgeomapfish/ ./docker-run make --makefile=travis.mk build'
+                sh 'travis/run-on ${HOME}/workspace/testgeomapfish/ ./docker-compose-run make update-po'
+                sh 'travis/run-on ${HOME}/workspace/testgeomapfish/ git add geoportal/testgeomapfish_geoportal/locale/*/LC_MESSAGES/testgeomapfish_geoportal-*.po'
+                sh 'travis/run-on ${HOME}/workspace/testgeomapfish/ git commit -m "Upgrade the po files"'
+                sh 'travis/run-on ${HOME}/workspace/testgeomapfish/ ./docker-run travis/empty-make --makefile=travis.mk help'
+                sh 'travis/run-on ${HOME}/workspace/testgeomapfish/ ./docker-run make --makefile=travis.mk build'
```

Comment:
I do not understand anything here, sorry.
Why do you have to make a commit ?
What does the travis/empty-make do ? no comments or description inside the script.
Can you add some comments for the other devs.

Reply:
Because after I test that the build didn't modify any files ant that we don't have any create files that's not in the git ignore, the po files modification should be accepted.